macros/drop_schemas_automation.sql

```diff
@@ -0,0 +1,30 @@
+{% macro drop_schemas_automation(drop_target_schema=true) %}
+    {{ return(adapter.dispatch('drop_schemas_automation')(drop_target_schema)) }}
+{%- endmacro %}
+
+{% macro default__drop_schemas_automation(drop_target_schema=true) %}
+
+{% set fetch_list_sql %}
+    {% if target.type not in ('databricks', 'spark') %}
+        select schema_name
+        from 
+        {{ quote(target.database) }}.INFORMATION_SCHEMA.SCHEMATA
+        where lower(schema_name) like '{{ target.schema | lower }}{%- if not drop_target_schema -%}_{%- endif -%}%'
+    {% else %}
+        SHOW SCHEMAS LIKE '{{ target.schema }}{%- if not drop_target_schema -%}_{%- endif -%}*'
+    {% endif %}
+{% endset %}
+
+{% set results = run_query(fetch_list_sql) %}
+
+{% if execute %}
+    {% set results_list = results.columns[0].values() %}
+{% else %}
+    {% set results_list = [] %}
+{% endif %}
+
+{% for schema_to_drop in results_list %}
+    {% do adapter.drop_schema(api.Relation.create(database=target.database, schema=schema_to_drop)) %}
```

Comment:
What are your thoughts on adding a cli message to let us know that the schema was successfully dropped?

Looking at the Shopify PR you opened, it is hard to see that this is even happening. I can confirm when looking at the data that his was successful, but it may be a better debugging experience to clearly call out in the logs that the schema was dropped. Thoughts?

Reply:
added a cli message!